spec/components/topic_view_spec.rb

```diff
@@ -3,6 +3,18 @@
 require 'rails_helper'
 require 'topic_view'
 
+def with_mega_topic_threshold(count, &blk)
+  original_const = Topic::MEGA_TOPIC_POSTS_COUNT
+  begin
+    Topic.send(:remove_const, "MEGA_TOPIC_POSTS_COUNT")
```

Comment:
I am not a fan of adding/removing consts in tests. Is there any way you could rely only on class methods and then stub them?